frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/SlideOut.java

```diff
@@ -173,35 +149,49 @@ private void slideOutBranch(String branchName) {
             (exceptionMessage == null ? "" : ": " + exceptionMessage);
         LOG.error(errorMessage);
         VcsNotifier.getInstance(project).notifyError(/* displayId */ null,
-            fmt(getString("action.GitMachete.BaseSlideOutAction.notification.title.slide-out-fail.HTML"), branchName),
+            getString("action.GitMachete.BaseSlideOutAction.notification.title.slide-out-fail.HTML").fmt(branchName),
             exceptionMessage == null ? "" : exceptionMessage);
       }
     });
   }
 
-  @UIThreadUnsafe
-  private @Nullable Boolean getDeleteLocalBranchOnSlideOutGitConfigValue(VirtualFile root) {
-    try {
-      val value = GitConfigUtil.getValue(project, root, DELETE_LOCAL_BRANCH_ON_SLIDE_OUT_GIT_CONFIG_KEY);
-      if (value != null) {
-        Boolean booleanValue = GitConfigUtil.getBooleanValue(value);
-        return booleanValue != null && booleanValue;
-      }
-    } catch (VcsException e) {
-      LOG.info(
-          "Attempt to get '${DELETE_LOCAL_BRANCH_ON_SLIDE_OUT_GIT_CONFIG_KEY}' git config value failed: key may not exist");
-    }
+  @ContinuesInBackground
+  private void getDeleteLocalBranchOnSlideOutGitConfigValueAndExecute(VirtualFile root,
```

Comment:
I've even thought of forcing that onto UI thread (with some `@IgnoreUIThreadUnsafeCalls` to circumvent our own QA)... but got [brutally stopped by IntelliJ itself](https://app.circleci.com/pipelines/github/VirtusLab/git-machete-intellij-plugin/8199/workflows/e8365e4c-2bb1-4237-8b7a-26f86cfd8f99/jobs/8625/steps): 

```
  [intellij-stdout] java.lang.Throwable: Assertion failed: Should not wait for built-in server on EDT
    [intellij-stdout] 	at com.intellij.openapi.diagnostic.Logger.assertTrue(Logger.java:254)
    [intellij-stdout] 	at org.jetbrains.ide.BuiltInServerManagerImpl.waitForStart(BuiltInServerManagerImpl.kt:97)
    [intellij-stdout] 	at git4idea.rebase.GitRebaseEditorService.getEditorCommand(GitRebaseEditorService.java:75)
    [intellij-stdout] 	at git4idea.rebase.GitHandlerRebaseEditorManager.prepareEditor(GitHandlerRebaseEditorManager.java:42)
    [intellij-stdout] 	at git4idea.rebase.GitHandlerRebaseEditorManager.lambda$prepareEditor$0(GitHandlerRebaseEditorManager.java:28)
    [intellij-stdout] 	at git4idea.GitUtil.tryRunOrClose(GitUtil.java:1034)
    [intellij-stdout] 	at git4idea.rebase.GitHandlerRebaseEditorManager.prepareEditor(GitHandlerRebaseEditorManager.java:27)
    [intellij-stdout] 	at git4idea.commands.GitImplBase.prepareGeneralPurposeEditor(GitImplBase.java:178)
    [intellij-stdout] 	at git4idea.commands.GitImplBase.run(GitImplBase.java:162)
    [intellij-stdout] 	at git4idea.commands.GitImplBase.runCommand(GitImplBase.java:63)
    [intellij-stdout] 	at git4idea.config.GitConfigUtil.getValue(GitConfigUtil.java:80)
    [intellij-stdout] 	at git4idea.config.GitConfigUtil.getValue(GitConfigUtil.java:73)
    [intellij-stdout] 	at com.virtuslab.gitmachete.frontend.actions.common.SlideOut.getDeleteLocalBranchOnSlideOutGitConfigValue(SlideOut.java:158)
    [intellij-stdout] 	at com.virtuslab.gitmachete.frontend.actions.common.SlideOut.run(SlideOut.java:76)
    [intellij-stdout] 	at com.virtuslab.gitmachete.frontend.actions.base.BaseSlideOutAction.doSlideOut(BaseSlideOutAction.java:78)
    [intellij-stdout] 	at com.virtuslab.gitmachete.frontend.actions.base.BaseSlideOutAction.actionPerformed(BaseSlideOutAction.java:58)
    [intellij-stdout] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    [intellij-stdout] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
    [intellij-stdout] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    [intellij-stdout] 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
....
```